File: SearchAndFindBackend/SearchAndFind.DTO/Category/CategoryResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SearchAndFind.DTO
{
    public class CategoryResponse:Response
    {
        public ICollection<CategoryDTO> Categories { get; set; }

        public CategoryResponse(string message) : base(message) { }

        public CategoryResponse()
        {
            Categories = new List<CategoryDTO>();
        }
    }
}

File: SearchAndFind-Frontend/app/src/main/java/uy/com/searchandfind/category/CategoryResponse.java
package uy.com.searchandfind.category;

import java.util.List;

import uy.com.searchandfind.Response;

/**
 * Created by strat on 23/10/2017.
 */

public class CategoryResponse extends Response {
    private CategoryDTO[] Categories;

    public CategoryResponse(String message){
        super(message);
    }

    public CategoryDTO[] getCategories() {
        return Categories;
    }

    public void setCategories(CategoryDTO[] categories) {
        Categories = categories;
    }
}
